### ISSUE TITLE
Accessibility: Lines read twice when navigating with up and down arrow in decompiler view

### ISSUE THREAD

**Issue by TheQuinbox** [NationalSecurityAgency/ghidra#6177]
**Describe the bug**
If I attempt to up/down arrow in the decompiler view (i.e. the text field containing the code of the current function), the line I focus gets read twice.
**To Reproduce**
1. Open Ghidra and start NVDA.
2. Load an executable, and fully analyze it.
3. Control+tab to the decompiler view.
4. Arrow up and down, and observe what the screen reader says.

**Expected behavior**
The line should be read only once per arrow press. Left and right arrow already do this. I assume this is because Ghidra tries to announce it to the screen reader when a line changes, but the screen reader already does this.
**Environment (please complete the following information):**
 - OS: Windows 10 22H2 (AMD64) build 19045.3803
 - Java Version: 21
 - Ghidra Version: 10.4 20230928
 - Ghidra Origin: Scoop

**Comment by ghidragon** [comment]
Yes, you are correct. I had it returning the line's text when ask for the current text. The interface that I had to implement also has the notion of returning a description of the thing your are on. For the description, I was returning the line number, followed by the line's text. Apparently, when you move up a line, the screen reader reads both the text and the description, causing it to repeat the line's text twice. I have changed the description to just report the line number. So now, when you move up or down a line in the decompiler, it will first read the text on that line, followed by the line number.

**Comment by TheQuinbox** [comment]
Awesome, thanks! I think this can be considered fixed then, provided it's been tested and works. I'm not sure how to test if it works, or I would.

**Comment by ghidragon** [comment]
Once this has been merged and pushed to github, this ticket will be closed. This will change will then be available at that point for those that use the bleeding edge of the master branch in github. Otherwise, it will be available in the next patch release.
